src/components/errorMessages.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
```

Comment:
It's recommended to use PascalCase for component file names. Rename the file to 'ErrorMessages.tsx'.